Dockerfile

```
@@ -0,0 +1,24 @@
+#Build: docker build -t jest .
+#Run: docker run --volume="$PWD:/usr/src/app" --rm jest:latest
+
+#Use Node 12.13.0 image
+#Includes Python2.7
+FROM node:12.13.0
```

```suggestion
FROM node:12
```

We shouldn't hard code a version here